tests/network-tests/src/flows/membership/creatingMemberships.ts

```diff
@@ -12,7 +12,6 @@ import { assert } from 'chai'
 export default async function membershipCreation({ api, env }: FlowProps): Promise<void> {
   const debug = extendDebug('flow:memberships')
   debug('Started')
-  api.enableDebugTxLogs()
```

Comment:
I'm actually enabling this log for all tests on `olympia` and it helped me debug a few issues in the past (perhaps it would also make debugging _this_ issue easier). This is only a little problematic when a runtime error is actually expected during the test.